web/src/app/modules/views/controller/modules/view-controller/services/view-controller.service.ts

```diff
@@ -56,6 +57,18 @@ export class ViewControllerService {
         return this._isEditorMaximized;
     }
 
+    public printEditor(): void {
+        this._isEditorPrint = true;
+    }
+
+    public unprintEditor(): void {
+        this._isEditorPrint = false;
+    }
```

Comment:
I think it would be better to combine the two methods above and pass a boolean parameter.